features/example/example.md

```diff
@@ -0,0 +1,25 @@
+# Example feature spec
+This contains an example of a new feature to be added.
+
+## Overview
+The purpose of this new feature is to be an example. It will modify the
+behavior of Nimbess in the following ways...
+
+## Functional Spec
+Indicate how your change will function. Describe in detail how it will work.
+
+## Code Changes
+Include a summary of the code changes necessary to implement your new
+functionality.
+
+## API/Schema changes
+Include any changes to the current API as well as database schema changes.
+
+## Performance Impact
+List any concerns around performance increase/decrease with your change.
+
+## User Impact
```

Comment:
external impacts or concerns?

Reply:
Please elaborate on this. Not sure what you mean.

Reply:
Do we want to add sections for external impacts like to other projects for example?

Reply:
OK will add that.